README.md

```diff
@@ -1,2 +1,20 @@
 # hw01
-Repository for homework 1
+
+![picture](PictureofMarie.jpg)
+
+# Marie's Biography
```

Comment:
Your headers are very well-executed and give the file a clean and concise feel. Great job!